wire-ios-sync-engine/Source/UserSession/ZMUserSession/ZMUserSession.swift

```diff
@@ -919,6 +952,10 @@ extension ZMUserSession: ZMSyncStateDelegate {
         }
     }
 
+    func trackAppOpenEvent() {
+        analyticsSession?.trackEvent(.appOpen)
+    }
```

Comment:
I think it would be better to forward the "app did become active" event to the user session, then it can track the event.

Reply:
Done: https://github.com/wireapp/wire-ios/commit/1241820bf2b275abc85fadbf21dc05bfa5d6d60c